infrastructure/smart-contracts/test/Referee.mjs

```diff
@@ -654,6 +680,50 @@ export function RefereeTests(deployInfrastructure) {
 			}
 		});
 
+		it("Check failure to submit a challenge with an invalid previous assertion id", async function () {
```

Comment:
Please rephrase this, instead of invalid we should specify whats invalid about the assertion

Reply:
rephrased